tn_api_v2_core.yml

```diff
@@ -0,0 +1,947 @@
+                    example: "gws-eJxNj8EOAiEMRD+GzL1ASdwbuKsxWcJBzRo8+P+f4VA17pQ2baa8QM45SIgS5JD3cni5xw1zWYCGwOzbFT6JCDynBhltnbez1uMKg0SfzPqHR7dFrtph6Yw29K398qTxIT"
+
+
+        DatasourcePriceBreakdownDetails:
+            type: object
+            description: Displays the pricing for each passenger type
+            properties:
+                passenger_type_code:
+                    type: string
+                    description: Passenger type code. Example = "ADT"
+                base_price:
+                    type: number
+                total_price:
+                    type: number
+                taxes:
+                    type: number
+                tax_breakdown:
+                    type: array
+                    items:
+                        $ref: '#/components/schemas/DatasourceTaxBreakdownDetails'
+
+        DatasourceTaxBreakdownDetails:
+            type: object
+            description: Breakdown of taxes amount into their categories
+            properties:
+                category:
+                    type: string
+                    description: Tax category. Example = "CA"
+                amount:
+                    type: number
+                    description: Tax amount. Example = 5.295
+
+        GenerateSolutionsResponse:
+            title: Generate Solutions Response
+            type: object
+            properties:
+                trip_id:
+                    type: string
+                    description: A unique key used for identification and when booking a trip.
+                    example: "fb3d8eff432a329dc21fad8449c686fa93e5ce24"
+                itineraries:
+                    $ref: '#/components/schemas/DatasourceItineraryInfo'
+
+        DatasourceItineraryInfo:
+            type: array
+            description: Array of itineraries sorted by weight.
+            items:
+                properties:
+                    weight:
+                        type: float
+                        description: Weight of the segment (time-value considered).
+                        example: 239
+                    total_price:
+                        type: float
+                        description: Price of the itinerary (fare and tax included).
+                        example: 197.1
+                    transportation_time:
+                        type: integer
+                        description: Total transportation time for the itinerary
+                        example: 425
+                    structure:
+                        type: Array of ints and/or Array of tuple/or Array of tuple and int
+                        description: |
+                            "Array representing the structure of the trip in terms of one-ways and open-jaw segments. 
+                            For example: [(0, 1), (3, 4), 2] means there would be three segments. One openjaw with key “01”, 
```

Comment:
means there would be three segments -> means there would be three bookings.

There are 5 segments in that structure, 3 bookings.

Reply:
will change!